tests/transformers/models/test_causal_lm_models.py

```diff
@@ -204,3 +206,30 @@ def test_causal_lm_export_with_deprecated_api(model_name):
     assert (
         new_api_ort_tokens == old_api_ort_tokens
     ).all(), "New API output does not match old API output for ONNX export function"
+
+
+@pytest.mark.causal_lm
+@pytest.mark.parametrize("model_name", test_models)
+def test_causal_lm(model_name):
+    """
+    Test function to validate the PyTorch model, the PyTorch model after KV changes, the ONNX model, and the Cloud AI 100 model, both with and without continuous batching.
+    ``Mandatory`` Args:
+        :model_name (str): Hugging Face Model Card name, Example: ``gpt2``
+    """
+    if model_name == "microsoft/Phi-3-mini-4k-instruct":
+        n_layer = 2  # test only 2 layer models
+    else:
+        n_layer = 1
+
+    check_causal_lm_pytorch_vs_kv_vs_ort_vs_ai100(model_name=model_name, n_layer=n_layer)
+
+
+@pytest.mark.causal_lm_prompt_len_1
+def test_causal_lm_pl_1():
```

Comment:
Please update to
```
@pytest.mark.on_qaic
@pytest.mark.parametrize("model_name", test_models)
def test_causal_lm_pytorch_vs_kv_vs_ort_vs_ai100_pl1(model_name):
```

Reply:
Yes  I have updated the marker here and name of the test function. I think making it parametrize is not a good idea which will test with all the models in `test_models` which will be time consuming.

Reply:
Yeah, sure. Let's not parametrize the PL=1 test.